### ISSUE TITLE
[BUG] TA-000013 flags Azure Functions with both identity types enabled

### ISSUE THREAD

**Issue by richardsondev** [Azure/template-analyzer#372]
### Describe the bug

[TA-000013](https://github.com/Azure/template-analyzer/blob/1ba73133c28786a16b2c19e5d5eef09eb2324538/docs/built-in-rules.md#ta-000013-managed-identity-should-be-used-in-your-function-app) is detecting Azure Functions as missing an identity when both System Assigned and User Assigned identities are attached with `type: 'SystemAssigned, UserAssigned'` however this is a valid value to enable identities.

### Expected behavior

TA-000013 should not detect Azure Functions as missing an identity when both SystemAssigned and UserAssigned are defined:

```bicep
  identity: {
    type: 'SystemAssigned, UserAssigned'
    userAssignedIdentities: {
      '${managedIdentity.id}': {}
    }
  }
```

### Reproduction Steps

1. Create a `Microsoft.Web/sites` resource in Bicep and define both SystemAssigned and UserAssigned identities to it:
```bicep


resource systemAndUserManagedWithSpaceIdentity 'Microsoft.Web/sites@2019-08-01' = {
  kind: 'functionapp'
  name: 'systemAndUserManagedWithSpaceIdentity'
  location: location
  properties: {
    siteConfig: {
      detailedErrorLoggingEnabled: false
      httpLoggingEnabled: false
      requestTracingEnabled: false
    }
  }
  identity: {
    type: 'SystemAssigned, UserAssigned'
    userAssignedIdentities: {
      '${managedIdentity.id}': {}
    }
  }
}
```
2. Run TemplateAnalyzer
3. The function is detected as missing an identity

**Comment by reynoldsa** [comment]
@richardsondev this is merged. Thanks for the contribution.
